### PR TITLE
fixed some problems, added new cover and README

### PR DESCRIPTION
**Pull request by kemamoto**
- Some fixes;
- Added new collection cover with cat:)
- Added readme;
- Some changes with console.log time and date;

### CODE REVIEW

File: MyClient.ts [hunk 1]
```diff
@@ -61,8 +32,8 @@ class MyClient {
 
   constructor(seed: string) {
     this.seed = seed;
-    this.collectionId = 735;
-    this.address = "5FNTBngp5E57ti1RYz7taHMChiqMvK2rQrSidH8nWwp1ALKW";
+    this.collectionId = -1;
+    this.address = "5FyM6ZTLNKQniseGKUwcqCL9hxoCGka62xV7ouTQ94afdqG5";
```
Comment (SergeyOkishev) [inline MyClient.ts]: we need to calculate it from seed, added to TODO

File: index.ts [hunk 2]
```diff
@@ -8,91 +8,43 @@ import {
...
-): Promise<NestedToken> {
-  const parent = await this.tokens.parent({ collectionId, tokenId });
-  if (parent)
-    return getTopmostParent.call(this, parent.collectionId, parent.tokenId);
-  const token = await this.tokens.get_new({ collectionId, tokenId });
-  if (!token) throw new Error("Token was not found");
-  return { ...token, nestingChildTokens: [] };
-}
+  
+  for(let i = 0; i < 1; i++) { // update 'i < 1' accordingly if you need to get more than 1 token
```
Comment (SergeyOkishev) [inline index.ts]: more than one collection, pehaps?

In that case it might be better to put into config(env)

File: index.ts [hunk 3]
```diff
@@ -8,91 +8,43 @@ import {
...
-      if (childChildren?.children?.length)
-        nestedChildToken.nestingChildTokens = await getChildTokens.call(
-          this,
-          childChildren,
-          depth + 1
-        );
-      return nestedChildToken;
-    })
-  );
+  // for(let i = 0; i < 1; i++) { // update 'i < 1' accordingly if you need to get more than 1 token
```
Comment (SergeyOkishev) [inline index.ts]: put depth/width in env as well and remove this commented part

File: index.ts [hunk 4]
```diff
@@ -8,91 +8,43 @@ import {
...
+  //   console.dir(res, { depth: 5 }); // to fix because this step returns undefined
+  //   console.log('The token has been created');
+  //   console.log('-------------------------')
+  // }
 
-  return childTokens;
-}
+  // for(let i = 0; i < 1; i++) { // update 'i < 1' accordingly if you need to get more than 1 token
+  //   const res = await MyClient.magic(4, 4);
+  //   console.dir(res, { depth: 5 }); // to fix because this step returns undefined
```
Comment (SergeyOkishev) [inline index.ts]: and this one

File: index.ts [hunk 5]
```diff
@@ -8,91 +8,43 @@ import {
...
-    throw new Error("Topmost token of a bundle has no nested tokens");
-  topMostParent.nestingChildTokens = await getChildTokens.call(
-    this,
-    topMostParentChildren
-  );
-  return topMostParent;
-}
+  // for(let i = 0; i < 1; i++) { // update 'i < 1' accordingly if you need to get more than 1 token
+  //   const res = await MyClient.magic(5, 5);
+  //   console.dir(res, { depth: 5 }); // to fix because this step returns undefined
```
Comment (SergeyOkishev) [inline index.ts]: and this one for sure

File: MyClient.ts (hunk 1 above)
Comment (SergeyOkishev) [reply MyClient.ts]: upd: required changes, so - while u are here, fix it too pls

**Review by SergeyOkishev — CHANGES_REQUESTED** [review]

File: MyClient.ts (hunk 1 above)
Comment (kemamoto) [reply MyClient.ts]: done

Comment (kemamoto) [reply MyClient.ts:36]: done

File: index.ts (hunk 2 above)
Comment (kemamoto) [reply index.ts]: done

File: index.ts (hunk 3 above)
Comment (kemamoto) [reply index.ts]: done

File: index.ts (hunk 4 above)
Comment (kemamoto) [reply index.ts]: done

File: index.ts (hunk 5 above)
Comment (kemamoto) [reply index.ts]: done

**Review by SergeyOkishev — APPROVED** [review]

File: MyClient.ts [hunk 6]
```diff
@@ -24,28 +25,25 @@ class MyClient {
...
       type: SignatureType.Sr25519,
     });
-
     await keyringProvider.init();
-
-    this.signer = keyringProvider.addSeed(this.seed).getSigner();
-
+    this.account = keyringProvider.addSeed(this.seed);
+    this.signer = this.account.getSigner();
+    this.address = this.account.instance.address;
```
Comment (SergeyOkishev) [inline MyClient.ts:46]: this.address = this.signer.getAddress()

**Review by SergeyOkishev — CHANGES_REQUESTED** [review]

**Pull request merged**